zebra-network/src/meta_addr.rs

```diff
@@ -248,6 +248,14 @@ pub enum MetaAddrChange {
         addr: SocketAddr,
         services: Option<PeerServices>,
     },
+
+    AddPeer {
```

Comment:
I was actually about to ask for a suggestion on this name. I couldn't come up with anything that would sound reasonable enough to me.

Reply:
I suggest `NewSeed`, because these peers come from the config or DNS seeders.

It might also be helpful to put this variant after the other `New*` variants.
And add a short doc line explaining where these peers come from.